javascript/javascript2/week1/index.js

```diff
@@ -44,7 +44,28 @@ console.log(getDanishLetter(danishString2, letters)); // returns {total: 4, æ:
 ==================================================*/
 
 const rootDiv = document.getElementById("root");
+const nameArea = document.getElementById("nameArea");
+const result = document.getElementById("result");
 
-function myFunction() {
-  etrName.innerHTML = "Helloooooo Jiu";
+function getUserName(e) {
+  console.log(1, nameArea.value != "");
+  if (nameArea.value != "") {
```

Comment:
This Comparison Operators `!= ""` will work but I suggest to use this one `!== ""`